backend/clubs/serializers.py

```diff
@@ -1409,10 +1409,14 @@ class UserClubVisitSerializer(serializers.ModelSerializer):
 
     person = serializers.HiddenField(default=serializers.CurrentUserDefault())
     club = ClubListSerializer(read_only=True)
+    ip = serializers.SerializerMethodField("get_ip")
+
+    def get_ip(self, obj):
+        return self.context["request"].META["REMOTE_ADDR"]
```

Comment:
This won't actually do anything to save the IP into the model, it'll just return the user their own IP in the response. You'll need to save the IP when you save the `ClubVisit` object, not when you fetch it.

Reply:
Ok great point not sure how I didn't notice this earlier